.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: node_js
+sudo: required
+env:
+  global:
+    - firebase_bucket=gs://dracco1993-tba-dev.appspot.com
+
+# branches:
+#   only:
+#   - master
+
+notifications:
+  email:
+    on_success: always
+    on_failure: always
```

Comment:
Once again, this might need to be changed on/before merge, to what we want it to be in `master.

Reply:
They go to whatever email is linked to the `GitHub` account that sets up the Travis integration.  You can change both of these setting in the Travis UI, so I've gone ahead and removed them from here.